src/app/navigation/navigation.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "src/assets/styles/colors.scss";
```

Comment:
Add style preprocessor option to `angular.json` for easier import

Reply:
yeah, great 👍